qa/rpc-tests/test_framework/nodemessages.py

```diff
@@ -1590,8 +1600,8 @@ class msg_headers(object):
     """
     command = b"headers"
 
-    def __init__(self):
-        self.headers = []
+    def __init__(self, headers = []):
+        self.headers = headers
```

Comment:
If I did self.headers.extend([1]) it would change every single object initialized with the default since all the headers lists are the same.  Instead this code should be:
__init__(self, headers = None):
self.headers = [] if headers is None else headers

Reply:
That is such an absurd language flaw. OK, I'll change it.

Reply:
Fixed